### PR TITLE
migrate code from googleapis/python-containeranalysis

### PR DESCRIPTION
**Pull request by msampathkumar**
- Container Analysis samples [(#2258)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2258)
- Update samples.py [(#2263)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2263)
- Adds updates including compute [(#2436)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2436)
- fix: Use different versions of pytest for python 2 and python3 [(#2558)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2558)
- Simplify noxfile setup. [(#2806)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2806)
- Update dependency mock to v4 [(#3216)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3216)
- [container_registry] fix: fix broken test [(#3436)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3436)
- Update dependency google-cloud-containeranalysis to v0.3.1 [(#3055)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3055)
- Update dependency google-cloud-pubsub to v1.4.2 [(#3340)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3340)
- [container analysis] effective severity [(#3478)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3478)
- [container_registry] fix: bump the pubsub timeout [(#3698)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3698)
- chore(deps): update dependency google-cloud-pubsub to v1.4.3 [(#3725)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3725)
- [container-registry] fix: mark a flaky test [(#3765)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3765)
- chore(deps): update dependency grafeas to v0.4.0 [(#3172)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3172)
- chore(deps): update dependency google-cloud-pubsub to v1.5.0 [(#3781)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3781)
- Replace GCLOUD_PROJECT with GOOGLE_CLOUD_PROJECT. [(#4022)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4022)
- Update dependency google-cloud-pubsub to v1.6.0 [(#4039)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4039)
- chore(deps): update dependency grafeas to v0.4.1 [(#4200)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4200)
- chore(deps): update dependency google-cloud-pubsub to v1.6.1 [(#4242)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4242)
- chore(deps): update dependency flaky to v3.7.0 [(#4263)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4263)
- chore(deps): update dependency pytest to v5.4.3 [(#4279)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4279)
- chore(deps): update dependency google-cloud-pubsub to v1.7.0 [(#4290)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4290)
- chore(deps): update dependency google-cloud-containeranalysis to v1 [(#4108)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4108)
- chore(deps): update dependency google-cloud-containeranalysis to v1.0.2 [(#4367)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4367)
- Update dependency pytest to v6 [(#4390)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4390)
- chore(deps): update dependency google-cloud-containeranalysis to v1.0.3 (#31)
- feat!: move to microgen (#33)
- chore(deps): update dependency google-cloud-containeranalysis to v2 (#41)
- chore(deps): update dependency grafeas to v1.0.1 (#40)
- chore(deps): update dependency google-cloud-pubsub to v2 (#46)
- chore(deps): update dependency google-cloud-containeranalysis to v2.1.0 (#56)
- chore(deps): update dependency google-cloud-pubsub to v2.2.0 (#61)
- chore(deps): update dependency mock to v4.0.3 (#67)
- chore(deps): update dependency google-cloud-containeranalysis to v2.2.0 (#76)
- chore(deps): update dependency google-cloud-containeranalysis to v2.2.1 (#91)
- chore(deps): update dependency google-cloud-pubsub to v2.3.0 (#88)
- chore(deps): update dependency google-cloud-pubsub to v2.4.0 (#94)
- fix: effective severity attribute error (#104)
- chore(deps): update dependency google-cloud-containeranalysis to v2.2.2 (#107)
- test: fix flaky test (#106)
- chore(deps): update dependency google-cloud-pubsub to v2.4.1 (#108)
- chore(deps): update dependency google-cloud-containeranalysis to v2.2.3 (#111)
- chore(deps): update dependency pytest to v6.2.3 (#115)
- chore(deps): update dependency pytest to v6.2.4 (#123)
- chore(deps): update dependency google-cloud-pubsub to v2.4.2 (#126)
- chore(deps): update dependency google-cloud-pubsub to v2.5.0 (#131)
- chore(deps): update dependency grafeas to v1.1.0 (#133)
- chore(deps): update dependency google-cloud-containeranalysis to v2.3.0 (#138)
- chore(deps): update dependency google-cloud-pubsub to v2.6.0 (#145)
- chore(deps): update dependency google-cloud-containeranalysis to v2.4.0 (#152)
- chore(deps): update dependency google-cloud-pubsub to v2.6.1 (#153)
- chore(deps): update dependency grafeas to v1.1.1 (#159)
- chore(deps): update dependency google-cloud-containeranalysis to v2.4.1 (#165)
- chore(deps): update dependency grafeas to v1.1.2 (#166)
- chore(deps): update dependency google-cloud-pubsub to v2.7.0 (#167)
- chore(deps): update dependency google-cloud-pubsub to v2.7.1 (#172)
- chore(deps): update dependency pytest to v6.2.5 (#176)
- chore(deps): update dependency google-cloud-pubsub to v2.8.0 (#179)
- chore(deps): update all dependencies (#186)
- chore(deps): update dependency grafeas to v1.1.4 (#188)
- chore(deps): update dependency grafeas to v1.2.0 (#195)
- chore(deps): update dependency google-cloud-containeranalysis to v2.5.0 (#196)
- chore(deps): update dependency grafeas to v1.3.0 (#199)
- chore(deps): update dependency google-cloud-containeranalysis to v2.6.0 (#200)
- chore(deps): update dependency google-cloud-containeranalysis to v2.6.1 (#205)
- chore(deps): update all dependencies (#208)
- chore(deps): update dependency grafeas to v1.4.0 (#209)
- chore(deps): update dependency google-cloud-pubsub to v2.9.0 (#213)
- chore(deps): update all dependencies (#227)
- chore(deps): update dependency google-cloud-containeranalysis to v2.6.3 (#230)
- chore(deps): update dependency google-cloud-containeranalysis to v2.7.0 (#238)
- chore(deps): update dependency pytest to v7 (#243)
- chore(deps): update dependency pytest to v7.0.1 (#245)
- chore(deps): update dependency google-cloud-containeranalysis to v2.7.1 (#246)
- chore(deps): update dependency grafeas to v1.4.2 (#249)
- chore(deps): update dependency google-cloud-pubsub to v2.10.0 (#255)
- chore(deps): update all dependencies (#256)
- chore(deps): update dependency google-cloud-containeranalysis to v2.7.3 (#258)
- chore(deps): update dependency google-cloud-pubsub to v2.11.0 (#259)
- chore(deps): update dependency pytest to v7.1.0 (#261)
- chore(deps): update dependency pytest to v7.1.1 (#262)
- chore(deps): update dependency google-cloud-pubsub to v2.12.0 (#274)
- chore(deps): update dependency pytest to v7.1.2 (#282)
- chore(deps): update all dependencies (#286)
- chore(deps): update dependency google-cloud-pubsub to v2.12.1 (#288)
- chore(deps): update all dependencies (#295)
- chore(deps): update all dependencies (#308)
- chore(deps): update dependency google-cloud-pubsub to v2.13.5 (#309)
- chore(deps): update dependency google-cloud-pubsub to v2.13.6 (#312)
- chore(deps): update dependency grafeas to v1.5.1 (#314)
- chore(deps): update dependency google-cloud-containeranalysis to v2.9.1 (#315)
- chore(deps): update dependency pytest to v7.1.3 (#323)
- chore(deps): update dependency google-cloud-pubsub to v2.13.7 (#329)
- chore(deps): update all dependencies (#333)
- chore(deps): update all dependencies (#336)
- chore(deps): update dependency google-cloud-pubsub to v2.13.9 (#337)
- chore(deps): update dependency google-cloud-pubsub to v2.13.10 (#338)
- chore(deps): update dependency pytest to v7.2.0 (#339)

## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

**Review by m-strzelczyk — APPROVED** [review]

File: container_registry/container_analysis/snippets/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google
```
Comment (dandhlee) [inline container_registry/container_analysis/snippets/README.md:1]: Could you move these files under `containeranalysis`?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -34,6 +34,7 @@
 /composer/**/*                         @leahecole @rachael-ds @rafalbiegacz @GoogleCloudPlatform/python-samples-reviewers
 /compute/**/*                          @m-strzelczyk @GoogleCloudPlatform/dee-infra @GoogleCloudPlatform/python-samples-reviewers
 /container/**/*                        @GoogleCloudPlatform/dee-platform-ops @GoogleCloudPlatform/python-samples-reviewers
+/container_analysis/**/*               @GoogleCloudPlatform/aap-dpes @GoogleCloudPlatform/python-samples-reviewers
```
Comment (dandhlee) [inline .github/CODEOWNERS:37]: I fixed this to be `/container_analysis` but should be `/containeranalysis/`.

**Review by dandhlee — CHANGES_REQUESTED** [review]

File: container_registry/containeranalysis/snippets/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google
```
Comment (dandhlee) [inline container_registry/containeranalysis/snippets/README.md:1]: Please remove `container_registry` directory and move the files under `containeranalysis`.

**Review by dandhlee — CHANGES_REQUESTED** [review]

**Review by dandhlee — APPROVED** [review]
LGTM

**Pull request merged**